eras/conway/test-suite/src/Test/Cardano/Ledger/Conway/Serialisation/Roundtrip.hs

```diff
@@ -1,26 +1,34 @@
 {-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 
 module Test.Cardano.Ledger.Conway.Serialisation.Roundtrip (allprops) where
 
+import Cardano.Ledger.Alonzo.TxWits (Redeemers)
+import Cardano.Ledger.Binary.Version (natVersion)
 import Cardano.Ledger.Conway.Genesis (ConwayGenesis (..))
 import Cardano.Ledger.Core (Era (..))
 import Data.Data (Proxy (..), typeRep)
-import Test.Cardano.Ledger.Binary.RoundTrip (roundTripCborExpectation)
+import Test.Cardano.Ledger.Binary.RoundTrip (
+  roundTripAnnExpectation',
+  roundTripCborExpectation,
+ )
 import Test.Cardano.Ledger.Conway.Serialisation.Generators ()
 import Test.Tasty (TestTree, testGroup)
 import Test.Tasty.QuickCheck (testProperty)
 
 allprops ::
   forall e.
-  ( Era e
-  ) =>
+  (Era e) =>
   TestTree
 allprops =
   testGroup
-    (show $ typeRep (Proxy @e))
-    [ testProperty "ConwayGenesis" $ roundTripCborExpectation @(ConwayGenesis (EraCrypto e))
+    ("Conway CBOR roundtrip " <> show (typeRep (Proxy @e)))
+    [ testProperty "ConwayGenesis" $
+        roundTripCborExpectation @(ConwayGenesis (EraCrypto e))
+    , testProperty "Redeemers" $
+        roundTripAnnExpectation' @(Redeemers e) (natVersion @9)
```

Comment:
Got it! I will use `roundTripAnnRangeExpectation` and add `maxBound` to it. However, as discussed on Slack, I was unsure if this test should have lower protocol versions as well, like: `(natVersion @2) maxBound`, as it fails for lower versions with the following error:
```
Encoder version: 2
Decoder version: 2
[...]
...expected size of 4, but found 3
```
...which is indicative of encoding version being 9 rather than 2?